docs/schemaform/form-config.md

```diff
@@ -196,6 +196,13 @@ In addition to the uiSchema options listed in the library [https://github.com/mo
     labels: {
       chapter30: 'A readable description (Chapter 30)'
     },
+
+    // This is a map of values to a component, some text, or some jsx. If your field
+    // is a radio widget, the content here will be shown underneath the radio button
+    // for that value when it is selected.
+    nestedContent: {
```

Comment:
I'm open to suggestions on how to unify those different approaches. The `view:descriptionBox` with an empty schema thing we've been doing feels hackish, so I'd love to get rid of it, but I don't have a better approach off hand for what we're using it for off hand.